tests/test_intensifier/test_hyperband_utils.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+from smac.intensifier.hyperband_utils import (
+    determine_HB,
+    determine_hyperband_for_multifidelity,
+    get_n_trials_for_hyperband_multifidelity,
+)
+
+
+def test_determine_HB():
+    min_budget = 1.0
+    max_budget = 81.0
+    eta = 3
+
+    result = determine_HB(min_budget=min_budget, max_budget=max_budget, eta=eta)
+
+    # Test table 1 from https://arxiv.org/pdf/1603.06560.pdf
```

Comment:
It should be noted that the this doesn't actually follow the *table* in the hyperband paper, but instead the *algorithm* (sadly they do not coincide, as noted in #977)

Reply:
updated comment